daml/Marketplace/Utils.daml

```diff
@@ -7,6 +7,20 @@ import DA.Next.Set
 import DA.Optional
 import DA.Text
 
+template DismissibleNotification
```

Comment:
perhaps this is better if it lives on its own module. We may end up adding more functionality in the future or make it a separate module that we import

Reply:
Agreed, I moved it to `Marketplace.Notification`.

Reply:
Unless you think `Marketplace.DismissibleNotification` would be better, I figured having it be more general would allow other types of notifications to be placed there.